implementations/rust/ockam/ockam_api/src/cli_state/cli_state.rs

```diff
@@ -41,6 +41,32 @@ pub struct CliState {
     database: SqlxDatabase,
     application_database: SqlxDatabase,
     tracing_enabled: TracingEnabled,
+    tx: Option<tokio::sync::broadcast::Sender<ChannelMessageType>>,
```

Comment:
This deserves a more explicit name than `tx` :-).
Also why not import `tokio::sync::broadcast` at the top of the file?

Reply:
done